src/data_management/rasters/RasterForm.js

```diff
@@ -659,539 +118,257 @@ class RasterFormModel extends Component {
-                      validateHours={this.validateHoursTemporalInterval}
-                      validateMinutes={this.validateMinutesTemporalInterval}
-                      validateSeconds={this.validateSecondsTemporalInterval}
-                    />
-                  </div>
-                ) : null}
-                {this.validateAll() ? (
-                  <div className={inputStyles.InputContainer}>
-                    <button
-                      type="button"
-                      className={`${buttonStyles.Button} ${buttonStyles.Success}`}
-                      style={{ marginTop: 10 }}
-                      onClick={() => {
-                        this.handleClickCreateRaster();
-                      }}
-                    >
-                      <FormattedMessage
-                        id="rasters.submit"
-                        defaultMessage="Submit"
-                      />
-                    </button>
-                  </div>
-                ) : (
-                  <div className={inputStyles.InputContainer}>
-                    <button
-                      type="button"
-                      className={`${buttonStyles.Button} ${buttonStyles.Inactive}`}
-                      style={{
-                        marginTop: 10,
-                        color: "grey",
-                        cursor: "not-allowed"
-                      }}
-                    >
-                      <FormattedMessage
-                        id="rasters.submit"
-                        defaultMessage="Submit"
-                      />
-                    </button>
-                    <span
-                      className={`${inputStyles.SubmitSpan}`}
-                      style={{ paddingLeft: 20, verticalAlign: "middle" }}
-                    >
-                      {this.generateInvalidMessage(currentStep)}
-                    </span>
-                  </div>
-                )}
-              </div>
-            </div>
-          </div>
-        </div>
+      {this.state.openOverlay ? (
+        <ErrorOverlay
+          isFetching={this.state.isFetching}
+          history={this.props.history}
+          errorMessage={this.state.modalErrorMessage}
+          handleClose={() =>
+            this.setState({ handlingDone: false, openOverlay: false })}
+          currentRaster={this.props.currentRaster || this.state.createdRaster}
+        />
+      ) : null}
+      <ManagementForm onSubmit={formData => this.onSubmit(formData, currentRaster)}
+                      wizardStyle={this.props.wizardStyle}
+      >
+
+        <SelectBox
+          name="selectedOrganisation"
+          title="Organisation"
+          subtitle="Specify which organisation owns this raster"
+          placeholder="Choose an organisation"
+          choices={this.props.organisations.available.map(organisation=>
+            [organisation.uuid.replace(/-/g, ""), organisation.name]
```

Comment:
If we remove the dashes here, then we don't also need to remove them on submit.

Reply:
I have them now at 3 places.
Maybe only the first place is needed:
- when retrieved from server
- when passed to GUI
- when written back to server

Let this sink in for a while..I will probably change

Reply:
removed all and tests seem to work.
API's seem to pass and accept dashes everywhere.